src/components/my-career/AddJobRole.jsx

```diff
@@ -50,72 +48,61 @@ const AddJobRole = ({ submitClickHandler }) => {
   };
 
   return (
-    <>
-      <Alert
-        variant="success"
-        show={isActivationAlertOpen}
-        onClose={closeActivationAlert}
-        className="mt-3"
-        dismissible
-      >
-        {LICENSE_ACTIVATION_MESSAGE}
-      </Alert>
```

Comment:
[inform] This `Alert` should _not_ be here as we risk showing duplicate alerts with it here (as was showing in the tests once tests were fixed to actually render the component and not throw JS errors...). My guess is that this was copy/pasta not properly removed when this component was first implemented.

By removing the `Alert`, we can remove the wrapping `React.Fragment` (`<>`).